### PR TITLE
DS-2638 update transaction meta added

### PR DESCRIPTION
**Pull request by mario7dujmovic**
https://jira.alldigitalrewards.com/browse/DS-2638

### CODE REVIEW

File: src/Transaction/AddTransactionMetaRequest.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    public function jsonSerialize()
+    {
+        if (empty($this->meta)) {
+            throw new \Exception('Transaction meta data has to be provided to request transaction meta update');
+        }
+
+        return [
+            "meta" => $this->meta ?? [],
```
Comment (zwalden) [inline src/Transaction/AddTransactionMetaRequest.php:56]: Should probably have a getter for meta which checks if it's set or not

Comment (zwalden) [reply src/Transaction/AddTransactionMetaRequest.php:56]: Well I suppose meta can never not be set, as it's in the constructor, so no need to have the ?? [] ternary null operator

**Review by zwalden — APPROVED** [review]

**Comment by zwalden** [comment]
Provided some comments, but really, this is OK as far as I can tell. 

When we merge this in, we'll want tag release and update your PR consuming this with a composer.json modification to match release

File: src/Transaction/AddTransactionMetaRequest.php (hunk 1 above)
Comment (mario7dujmovic) [reply src/Transaction/AddTransactionMetaRequest.php]: @zwalden i got rid of the ?? [] just because why not!

**Review by jmuto2 — APPROVED** [review]

**Pull request merged**